yb-voyager/cmd/endMigrationCommand.go

```diff
@@ -379,25 +402,25 @@ func cleanupFallBackDB(msr *metadb.MigrationStatusRecord) {
 		return
 	}
 
-	utils.PrintAndLog("cleaning up voyager state from fallback db...")
+	utils.PrintAndLog("cleaning up voyager state from source db(used for fall-back)...")
```

Comment:
Can you maybe share a screenshot of all the things that we print on running this command?

Reply:
@shubham-yb can you share that with Aneesh when you are testing the fallback part.